Gemfile

```diff
@@ -9,8 +9,8 @@ ruby RUBY_VERSION
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem "jekyll", "3.3.0"
-
+gem "jekyll", ">= 3.6.3"
+i
```

Comment:
Possibly a typo by switching to insert mode while already being in insert mode? :D